cmd/metadata.go

```diff
@@ -0,0 +1,72 @@
+package cmd
```

Comment:
rename this to `fake_metadata.go` to make it clear it's a test fake

Reply:
Sure , I'll rename it.